src/GovUk.Education.ExploreEducationStatistics.Publisher.Model/ViewModels/CachedReleaseViewModel.cs

```diff
@@ -1,96 +1,54 @@
-﻿using System;
+﻿#nullable enable
+using System;
 using System.Collections.Generic;
 using GovUk.Education.ExploreEducationStatistics.Common.Model;
 
 namespace GovUk.Education.ExploreEducationStatistics.Publisher.Model.ViewModels
 {
-    public class CachedReleaseViewModel
+    public record CachedReleaseViewModel
     {
-        public CachedReleaseViewModel()
-        {
-        }
-
-        protected CachedReleaseViewModel(
-            Guid id,
-            string title,
-            string yearTitle,
-            string coverageTitle,
-            string releaseName,
-            PartialDate nextReleaseDate,
-            DateTime? published,
-            string slug,
-            ReleaseTypeViewModel type,
-            List<ReleaseNoteViewModel> updates,
-            List<ContentSectionViewModel> content,
-            ContentSectionViewModel summarySection,
-            ContentSectionViewModel headlinesSection,
-            ContentSectionViewModel keyStatisticsSection,
-            ContentSectionViewModel keyStatisticsSecondarySection,
-            List<FileInfo> downloadFiles,
-            string dataGuidance,
-            string preReleaseAccessList,
-            List<LinkViewModel> relatedInformation,
-            DateTime? dataLastPublished)
+        public CachedReleaseViewModel(Guid id)
```

Comment:
Could there be any risk here of things that are now always initialised by default that were previously null changing the behaviour at all for any of these fields? The changes do all look ok to me and I suppose if the tests for newly published Releases are ok then we should be fine for older Releases when we come to regenerate all the cached content next time. Just conscious that a complete refresh might occur much later down the line.

Reply:
The primary fields like `Title`, `YearTitle`, `CoverageTitle`, `ReleaseName` and `Slug` should all be set when a release is made, so declaring their defaults as empty string is mostly just to satisfying the nullable constraints.

For more 'optional' fields like `DataGuidance` and `PreReleaseAccessList`, we actually define these as non-optional strings in the frontend and already account for the possibility that they are non-null, but empty.

I've swapped the `ContentSectionViewModel` fields (e.g. summary, headlines, etc) back to non-nullable as it turns out we actually create all of these when a release is created. I'd mistakenly assumed these were optional because admin users don't technically need to create any blocks in any of these sections.

Reply:
Cool, thanks for checking!